packages/nimble-components/src/icon-base/tests/icon-metadata.ts

```diff
@@ -100,15 +109,30 @@ export const iconMetadata: {
     IconBookMagnifyingGlass: {
         tags: ['analyze']
     },
-    IconCalendar: {
-        tags: ['schedule']
-    },
     IconCalendarCheckLines: {
         tags: ['test', 'plan', 'schedule']
     },
+    IconCalendarClock: {
+        tags: ['time', 'schedule']
+    },
     IconCalendarDay: {
         tags: ['date', 'time', 'schedule']
     },
+    IconCalendarDays: {
+        tags: ['schedule']
+    },
```

Comment:
Are there any tags to differentiate usage of `day` vs `days`?

Reply:
Paging Dr. @NIbokeefe

Reply:
Not currently.  Right now I've only gotten a request for calendar "day." And the closest related one is "today"